test/recipes/80-test_cmp_http.t

```diff
@@ -217,8 +216,24 @@ indir data_dir() => sub {
                     test_cmp_http_aspect($server_name, $aspect, $tests);
                 };
             };
-            stop_server($server_name, $pid) if $pid;
-            ok(1, "$server_name server has terminated");
+
+            if ($server_name eq "Mock") {
+                stop_server($server_name, $pid) if $pid;
+                ok(1, "$server_name server has terminated");
+
+                if (-s $faillog) {
+                    indir "Mock" => sub {
+                        print STDERR "$server_name STDERR output is:\n";
+                        if (open F, $Mock_serverlog) {
+                            while (<F>) {
+                                print STDERR $_;
+                            }
+                            close F;
+                        }
+                        print STDERR "$server_name STDERR output end.\n";
```

Comment:
I would recommend to output everything to STDOUT.
And furthermore prefix each line by comment sign `# ` so it cannot
accidentally contain keywords, like "ok" or "not ok" etc.
The original issue that I wanted to fix here, was output corruption
because STDOUT and STDERR are independet file streams,
which are usually buffered, and are therefore flushed when the
buffer is full, which is typically not at the end of line.
This caused in very rare cases a TAP parser error.
So by sending all output to the same file stream this is more safe.

Reply:
Good points -done.